worth2/selftalk/models.py

```diff
@@ -0,0 +1,199 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.db import models
+from ordered_model.models import OrderedModel
+
+from worth2.main.generic.models import BasePageBlock
+
+
+class Statement(OrderedModel):
+    """A negative statement.
+
+    Each Statement has multiple Refutations for the participant to pick
+    from.
+    """
+
+    text = models.TextField()
+    other_text = models.TextField(blank=True, null=True)
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+
+class Refutation(OrderedModel):
+    """A Refutation is an option used to refute a negative statement.
+
+    Each Refutation is tied to a Statement.
+    """
+
+    statement = models.ForeignKey(Statement)
+    text = models.TextField()
+    other_text = models.TextField(blank=True, null=True)
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+
+class StatementBlock(BasePageBlock):
+    """A PageBlock for choosing negative statements to refute.
+
+    This block can be either "internal" or "external". Internal refers to
+    "My Self Talk", where the subject of the negative statements is the
+    user themselves. "External" indicated that this is a scenario based
+    around a video, rather than a self-reflective excercise.
+
+    If this block is "external", it gives the participant a list of
+    statements to check off what they remember hearing in the video they
+    just watched.
+
+    The statements the participant chooses will populate the fields in
+    the next page, which will be a RefutationBlock.
+    """
+
+    display_name = 'Self-Talk Negative Statement Block'
+    template_file = 'selftalk/do_you_remember_block.html'
+
+    statements = models.ManyToManyField(Statement)
+    subject_name = models.TextField(
```

Comment:
Minor, but worth noting: I might break this out into 2 fields, one a "choice" field with predefined values and a free-form subject field.